deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+ENCRYPTED_KEY_VAR="encrypted_${ENCRYPTION_LABEL}_key"
+ENCRYPTED_IV_VAR="encrypted_${ENCRYPTION_LABEL}_iv"
+ENCRYPTED_KEY=${!ENCRYPTED_KEY_VAR}
+ENCRYPTED_IV=${!ENCRYPTED_IV_VAR}
+openssl aes-256-cbc -K $ENCRYPTED_KEY -iv $ENCRYPTED_IV -in deploy_key.enc -out deploy_key -d
+chmod 600 deploy_key
+eval `ssh-agent -s`
+ssh-add deploy_key
+echo "$SERVER $SERVER_PUBLIC_KEY" > known_hosts
+rsync -avz -e "ssh -o UserKnownHostsFile=known_hosts" src/ $DEPLOY_USER@$SERVER:$WEB_ROOT/
```

Comment:
Nit: expanding command line flags makes things easier for future readers. So maybe

--archive --verbose --compress --rsh="ssh ..."